lib/ar_state_machine.rb

```diff
@@ -179,10 +194,8 @@ def setup(states)
         define_method "make_#{ss}" do |last_edited_by_override = nil|
           self.state = ss
           self.last_edited_by_id = last_edited_by_override if last_edited_by_override
+          self.set_state_by_id()
```

Comment:
() is not necessary. this is ruby

Reply:
kevin and I both felt it looked really weird without the (), but if you want it, I'll remove it.